src/main/resources/i18n/messages.properties

```diff
@@ -534,13 +534,16 @@ UserDetailsPage.notification.success=User updated successfully.
 UserDetailsPage.notification.error=An error has occurred while updating the user details. Please check all the fields before updating.
 UserDetailsPage.form.firstName.label=First Name
 UserDetailsPage.form.firstName.required=A first name is required.
+UserDetailsPage.form.firstName.min=Must be at least 2 characters long.
```

Comment:
Deep is updating the password reset page.  His will be in before yours so we should take a look at these again once his PR is in and see if we cannot find common translations - since it is essentially the same thing

Reply:
@joshsadam did his PR get merged in?